packages/govern-console/src/containers/DAO/DaoMainPage.tsx

```diff
@@ -64,7 +64,7 @@ const DaoMainPage: React.FC<{
   const [fetchMoreProposal, updateFetchMoreProposals] = useState<any>();
   const [isProfilePage, setProfilePage] = useState(false);
   const [daoDetails, updateDaoDetails] = useState<any>();
-  const searchString = useRef('');
+  const [searchString, setSearchString] = useState('');
```

Comment:
@Rekard0  can we have a proper name ? what is `searchString` ? what does it search? if it's dao name, `searchDao` should be the name.